gen/templates/resource.go.tmpl

```diff
@@ -30,6 +31,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
 	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
 	{{- if or .HasChild .HasBitmask}}
+	"github.com/hashicorp/terraform-plugin-framework/attr"
```

Comment:
Make just to run `go mod tidy` & `go mod vendor` to pick up any packages required for the build.